frontend/discourse/admin/controllers/admin-config/login.js

```diff
@@ -0,0 +1,29 @@
+import Controller from "@ember/controller";
+import { service } from "@ember/service";
+import { i18n } from "discourse-i18n";
+
+export default class AdminConfigLoginController extends Controller {
+  @service router;
+
+  get currentTabLabel() {
```

Comment:
```suggestion
  get currentBreadcrumbLabel() {
```

To differentiate between the tabs that show on the page and the breadcrumbs